packages/web-react/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
```

Comment:
Do we want to use `prop-types` instead of Typescript as default? There is no product using `prop-types` for typing.  We are using Typescript as default. So we should consider using Typescript also here. I know we must compile it then but I think we should compile our code anyway to support and satisfy multiple ES versions and serverside rendering.

Reply:
Transition to TS is to be fixed during #7.